HackYourFuture-CPH/Guide-IT
# Final question button functionality

# Description

This makes the button link to the results page and submit the users answer tot he final question tot he database.

Fixes # 137 and 172

There is an error in issue 172. It states that the URL should ens with "/resultPage". But the router says it should be "/quiz-results/:userId".

# How to test?

Open branch and run "npm run dev". Follow the quiz through to the results page. Make sure your database has up to date migrations and seeds before beginning.

# Checklist
